pkg/cri/resource-manager/cache/cache.go

```diff
@@ -497,7 +503,7 @@ type Cachable interface {
 // itself upon startup.
 type Cache interface {
 	// InsertPod inserts a pod into the cache, using a runtime request or reply.
-	InsertPod(id string, msg interface{}) Pod
+	InsertPod(id string, msg interface{}, info *PodStatus) Pod
```

Comment:
Slightly different parameter naming in the interface and function declaration:
`InsertPod(id, msg, info)`
vs
`InsertPod(id, msg, status)`

Maybe the status would be better, because otherwise it would be tempting to consider `s/PodStatus/PodInfo`.

Reply:
Fixed.